third_party/meerkat/Component/mmSH/android/java/src/com/samsung/android/meerkat/MeerkatServerService.java

```diff
@@ -196,6 +200,15 @@ private static boolean shouldUseDebugIni() {
     }
 
     public static boolean startCastanetsRenderer(String args) {
+        // Launch OffloadService directly.
+        if (args.matches("(^|.*\\s)--type=offloadworker($|\\s.*)")) {
```

Comment:
Is the '\\s' correct regex?
Isn't the '\s' enough to check if there is space before and after?

Reply:
> Is the '\s' correct regex?
> Isn't the '\s' enough to check if there is space before and after?

No, It is not matched only with "\s" without ". *".
I also finally found that ". *" needed to check a space before and after.

Reply:
Yes, I mean, Isn't "(^|.*\s) .... ($|\s.*)" enough? instead of the '\\s'

Reply:
Java escape character is backslash('\'). So it needs double backslash for backslash, not escape.

Reply:
I see. Thank you for your kind explanation.